### PR TITLE
Feature/room

### PR DESCRIPTION
**Pull request by souziel**

### CODE REVIEW

File: client/public/locales/fr.json [hunk 1]
```diff
@@ -270,5 +271,20 @@
...
     "close": "Fermer",
     "unlock": "Déverouiller"
+    },
+  "rooms": {
+    "myRooms": "Mes pièces",
+    "search": "Rechercher ...",
+    "createRoom": "Nouvelle pièce",
+    "roomName": "Nom de la pièce",
+    "roomDevices": "Equipement dans la pièce",
+    "roomCreated": "Pièce sauvegardé avec sucess",
```
Comment (guyts96) [inline client/public/locales/fr.json]:
```suggestion
    "roomCreated": "Pièce sauvegardée avec succès",
```

File: client/public/locales/fr.json [hunk 2]
```diff
@@ -270,5 +271,20 @@
...
+    "roomDeleted": "Pièce supprimé avec sucess"
```
Comment (guyts96) [inline client/public/locales/fr.json]:
```suggestion
    "roomDeleted": "Pièce supprimée avec succès"
```

File: client/src/components/rooms/RoomDetail.svelte [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    >
+        {currentRoom.roomId !== '' ? currentRoom.name : tr('rooms.createRoom')}
+    </h1>
+    <form class="mb-4" on:submit={() => false}>
+        <div class="flex flex-col mb-4">
+            <label
+                class="mb-2 font-bold text-lg text-grey-darkest"
+                for="Name"
+            >{tr('rooms.roomName')}</label>
+            <input
```
Comment (guyts96) [inline client/src/components/rooms/RoomDetail.svelte]: Maybe use the InputBar component ?

File: client/src/components/rooms/RoomComponent.svelte [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            on:click={() => {
+                RoomService.deleteRoom(room);
+            }}
+            iconPath="icons/button/trash.svg"
+        />
+    </div>
+</div>
+
+<style>
+    #group {
```
Comment (asoran) [inline client/src/components/rooms/RoomComponent.svelte]: "group" ? not "room" ? :p

**Review by asoran — APPROVED** [review]

**Review by guyts96 — APPROVED** [review]

**Pull request merged**